app/scripts/constants.js

```diff
@@ -255,6 +255,23 @@ angular.extend(window.OPENSHIFT_CONSTANTS, {
                 "/create-route",
                 "/edit/routes/"
               ]
+            },
+            {
+              label: "Provisioned Services",
+              href: "/browse/service-instances",
+              prefixes: [
+                "/browse/service-instances/"
+              ],
+              isValid: function(){
+                return _.get(window.OPENSHIFT_CONSTANTS, 'ENABLE_TECH_PREVIEW_FEATURE.service_catalog_landing_page');
+              },
```

Comment:
Should be able to remove `isValid` now that #2078 is in the merge queue. The `canI` check below should hide the menu item when service catalog is not enabled.

Reply:
#2078 has merged